### PR TITLE
`str_split` returns an empty array since PHP 8.2

### PR DESCRIPTION
**Pull request by herndlm**
Refs: https://github.com/php/php-src/pull/8945

See also https://3v4l.org/XfZO4 and https://3v4l.org/XfZO4/rfc#vgit.master

I never touched the delta files, is this correct like this? One of the tests at least seems to verify it. And the comments in that file are obviously outdated, but I did not touch them because they're like that in other delta files as well.

UPDATE: sorry, was doing too many things at once and forgot to cleanly run make again.. I thought I could avoid nested ternaries with the match expression 😅

### CODE REVIEW

**Comment by ondrejmirtes** [comment]
Perfect, thank you :)

File: resources/functionMap_php82delta.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php // phpcs:ignoreFile
+
+/**
+ * Copied over from https://github.com/phan/phan/blob/8866d6b98be94b37996390da226e8c4befea29aa/src/Phan/Language/Internal/FunctionSignatureMap_php80_delta.php
+ * Copyright (c) 2015 Rasmus Lerdorf
+ * Copyright (c) 2015 Andrew Morrison
+ */
+
+/**
+ * This contains the information needed to convert the function signatures for php 8.0 to php 7.4 (and vice versa)
```
Comment (BackEndTea) [inline resources/functionMap_php82delta.php:10]: I think this should be 8.2 right?

And is this the difference between 7.4 and 8.2, or between 8.1 and 8.2?

Comment (herndlm) [reply resources/functionMap_php82delta.php:10]: Yeah, see PR description :) UPDATE: and it is the difference between 8.1 and 8.2, yes.

Comment (herndlm) [reply resources/functionMap_php82delta.php:10]: @ondrejmirtes should the delta files all (except the original one where this was copied over) updated in that regard?

Comment (ondrejmirtes) [reply resources/functionMap_php82delta.php:10]: I don't care, I never read that comment :) The filename says it all.

**Pull request merged**